index.js

```diff
@@ -115,9 +115,12 @@ class ForwardEmail {
 
   parseUsername(address) {
     ({ address } = addressParser(address)[0]);
-    return address.indexOf('+') === -1
+    username = address.indexOf('+') === -1
```

Comment:
Wouldn't this want to be `let username = ...` so as not to introduce a new global variable `"username"`?

Reply:
yes, I will resubmit.